library/agent/context/markUnsafe.ts

```diff
@@ -0,0 +1,32 @@
+import { getInstance } from "../AgentSingleton";
+import { updateContext } from "../Context";
+import { ContextStorage } from "./ContextStorage";
+
+export function markUnsafe(payload: unknown) {
+  const agent = getInstance();
+
+  if (!agent) {
+    return;
+  }
+
+  const context = ContextStorage.getStore();
+
+  if (!context) {
+    return;
```

Comment:
If the user calls this method outside of the context of an incoming http request, he might never notice that it doesn't have any effect to call the method?